pkg/controller/common/rollout/rollout_plan_controller.go

```diff
@@ -351,7 +351,7 @@ func (r *Controller) GetWorkloadController() (workloads.WorkloadController, erro
 		if r.targetWorkload.GetKind() == reflect.TypeOf(apps.Deployment{}).Name() {
 			// check whether current rollout plan is for workload rolling or scaling
 			if r.sourceWorkload != nil {
-				return workloads.NewDeploymentController(r.client, r.recorder, r.parentController,
+				return workloads.NewDeploymentRolloutController(r.client, r.recorder, r.parentController,
```

Comment:
Library `workloads` comes from "github.com/oam-dev/kubevela/pkg/controller/common/rollout/workloads", which indicates, DeploymentController is one rollout controller. Why do we have to change the name? Could you please explain it a little bit more?

Reply:
`DeploymentController` could be `DeploymentScaleController` or `DeploymentRolloutController`. Since we name the constructor of `DeploymentScaleController` as [NewDeploymentScaleController](https://github.com/oam-dev/kubevela/blob/ed4844d518a1718aefaccbaba979911478bbe233/pkg/controller/common/rollout/workloads/deployment_scale_controller.go#L47), it would be more clear to rename `DeploymentRolloutController` constructor to `NewDeploymentRolloutController`.